hack/crdgen.sh

```diff
@@ -8,6 +8,11 @@ add_header() {
   mv tmp "$1"
 }
 
+if [ "$(ls -t manifests/base/crds/full/*.yaml | head -n1)" -nt "$(ls -t pkg/apis/workflow/v1alpha1/*.go | grep -v 'deepcopy\|generated\|test'| head -n1)" ]; then
```

Comment:
this pattern I use basically says:

if [ "the newest CRD file" is newer than "the newest source file"  ]